app/Controllers/Sales.php

```
@@ -1463,27 +1499,33 @@ public function restore(int $sale_id = NEW_ENTRY, bool $update_inventory = true)
      * @return ResponseInterface
      * @throws ReflectionException
      */
-    public function postSave(int $sale_id = NEW_ENTRY): ResponseInterface
+    public function postSave(int $saleId = NEW_ENTRY): ResponseInterface
```

PHPStan (2.2.7)

[error] PHPDoc tag `@param` references unknown parameter: $sale_id


(parameter.notFound)